flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/ApplicationStatus.java

```diff
@@ -32,7 +32,7 @@
 	FAILED(1443),
```

Comment:
I understand your reasoning. In the case of a job cluster this might be problematic though. We might want to revisit job cluster life cycle and exit codes in general.